addons/block_code/ui/util.gd

```diff
@@ -0,0 +1,6 @@
+extends Node
```

Comment:
I think you should extend `Object` if you're just going to put static functions in here. Also, I think you might as well add this first in the series and then use it in tooltip.gd.

Reply:
OK